composer.json

```
@@ -9,9 +9,9 @@
     "type": "library",
     "license": "MIT",
     "require": {
-        "php": "^8.0",
+        "php": "^8.1",
         "codeception/codeception": "^5.0",
-        "codeception/lib-innerbrowser": "^3.0",
+        "codeception/lib-innerbrowser": "^4.0",
```

I don't see any reason to drop PHP 8.0 support for now. With this we can keep full backward compatibility and no need for breaking change.

```suggestion
        "codeception/lib-innerbrowser": "^3.0 || ^4.0",
```